src/chigame/games/views.py

```diff
@@ -91,6 +91,26 @@ def test_func(self):
         return self.request.user.is_staff
 
 
+def search_results(request):
+    query = request.GET.get("query")
+
+    """
+    The Q object is an object used to encapsulate a collection of keyword
+    arguments that can be combined with logical operators (&, |, ~) which
+    allows for more advanced searches. More info can be found here at
+    https://docs.djangoproject.com/en/4.2/topics/db/queries/#complex-lookups-with-q-objects
+    """
+    object_list = Game.objects.filter(
+        Q(name__icontains=query)
+        | Q(category__name__icontains=query)
```

Comment:
This might need an update after PR #236 when "category" becomes "categories".

Reply:
#236 has been merged!